src/libPMacc/include/math/vector/Vector.hpp

```diff
@@ -119,6 +120,7 @@ struct Vector : private T_Storage<T_Type, T_dim>, protected T_Accessor, protecte
     typedef T_Accessor Accessor;
     typedef T_Navigator Navigator;
     typedef Vector<type, dim, Accessor, Navigator, T_Storage> This;
+    typedef typename boost::call_traits<type>::param_type paramType;
```

Comment:
Please use upper case letter for types. `ParamType`
Maybe `OptimalParamType` is a more meaningful name.